src/interface/swarm_container.cpp

```diff
@@ -74,6 +74,24 @@ void SwarmContainer::Remove(const std::string &label) {
   swarmMap_.erase(label);
 }
 
+TaskStatus SwarmContainer::Defrag(double min_occupancy) {
+  Kokkos::Profiling::pushRegion("Task_SwarmContainer_Defrag");
+  PARTHENON_REQUIRE_THROWS(min_occupancy >= 0. && min_occupancy <= 1.,
+                           "Max fractional occupancy of swarm must be >= 0 and <= 1");
+
+  for (auto &s : swarmVector_) {
+    s->SetupPersistentMPI();
+    if (s->GetNumActive() > 0 &&
+        s->GetNumActive() / (s->GetMaxActiveIndex() + 1) < min_occupancy) {
```

Comment:
Does this work as intended?
`s->GetNumActive() / (s->GetMaxActiveIndex() + 1)` is integer arithmetic, isn't it?

Reply:
Whoa nice catch! Yes this was broken. I fixed it (`1` -> `1.0`) and checked the result with `printf`s.

Reply:
one thumbs up isnt enough :+1: :+1: